pmix_governance.tex

```diff
@@ -425,6 +425,58 @@ \subsubsection{ASC Secretaries}\label{asc-secretaries}}
 Alternate for a Member may vote on behalf of the Member subject to the
 one vote per Member limit.
 
+\hypertarget{asc-guidance}{%
+\subsection{ASC Guidance}\label{asc-guidance}}
+
+This section provides general guidance for the PMIx ASC and participants.
+The rationale behind providing this guidance is to outline
+philosophy and best practices the ASC should follow in 
+maintaining the PMIx Standard. 
+
+\hypertarget{release-schedule-guidance}{%
+\subsubsection{Release Schedule Guidance}\label{release-schedule-guidance}}
+
+As described in Section~\ref{release-managers},
+the schedule of major releases of the PMIx
+Standard is determined by the Release Managers in coordination with other
+members of the ASC. There is no prescribed major release schedule that the
+Release Managers must follow. However, the Release Managers will prioritize
+timely releases of substantive changes to the Stable entries in the 
+PMIx Standard. Substantive changes
+include: deprecated or removed
+APIs, keys, or other text in the standard; alterations of APIs or keys; and
+alterations of text that result in changed meaning or clarifications. The
+Release Managers will prioritize major releases of the standard over waiting for
+additional changes that are in progress but not yet accepted by the ASC.
+
+\hypertarget{administrative-changes-guidance}{%
+\subsubsection{Minor Document Changes Guidance}\label{administrative-changes-guidance}}
+The intention of the rigorous process outlined in 
+Section~\ref{the-standardization-process} for proposed
+changes to the PMIx Standard is to ensure quality and consistency of 
+the interface and the overall document. However, in some cases, 
+PMIx participants may propose minor changes to the document that 
+should not require the full process required for substantive changes. 
+Examples of these minor changes include fixes for typographical errors 
+(``typos''), missing or erroneous punctuation, and formatting changes.
+
+In the case minor changes are proposed, the proposer should 
+create an issue and pull request as described in Sections~\ref{initial-discussion}
+and \ref{proposed-modification} and then bring the
+changes to the attention of the ASC Co-Chairs and Release Managers.
+The Co-Chairs and Release Managers will evaluate the proposed

```

Comment:
@SteVwonder says: Do we want to allow community members (or ASC members) to also chime in with concern over how substantial a change is? Or keep it limited to Co-Chairs and RMs?

Reply:
@SteVwonder I added some text to clarify that everyone should look at it and comment. Does it address your concern?

Reply:
LGTM!